COLLABORATOR_GUIDE.md

```diff
@@ -93,14 +93,17 @@ information regarding the change process:
 
 - A `Reviewed-By: Name <email>` line for yourself and any
   other Collaborators who have reviewed the change.
-- A `PR-URL:` line that references the full GitHub URL of the original
+- A `PR-URL:` line that references the *full* GitHub URL of the original
   pull request being merged so it's easy to trace a commit back to the
   conversation that led up to that change.
-- A `Fixes: X` line, where _X_ is either includes the full GitHub URL
+- A `Fixes: X` line, where _X_ is either includes the *full* GitHub URL
```

Comment:
While you're in here, might as well fix the `is either includes` business, yeah?